src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -343,7 +343,11 @@ internal override void RequestRun()
                 // Make this RunSettings.RunEnabled private, introduce the new flag and remove the "executingTask" variable. 
                 if (RunSettings.RunEnabled || executingTask)
                 {
-                    Run();
+                    // Skip's the execution, if packing loading is not completed. 
```

Comment:
`skip the execution if runs have been disabled - currently this flag is only set by the Package Loader`